src/components/tasks/card/card.module.scss

```diff
@@ -97,7 +100,46 @@ $black: #252525;
 .progressYellow {
   background-color: $yellow;
 }
+
 .editButton {
   text-align: left;
   cursor: pointer;
 }
+
+.dropdownMenu {
+  position: absolute;
+  right: 0;
+  top: 90%;
+  z-index: 10;
+}
+
+.inputAssigneeSearch {
+  margin-bottom: .5rem;
+}
+
+.assigneeMenu {
+  border-radius: .25rem;
+  background-color: #ffffff;
```

Comment:
Please use `constants` for defining colors.

Reply:
okay